src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
```

Comment:
```suggestion
```

Reply:
Let's delete the odd instructions

Reply:
perfectionists are evil people)

Reply:
Just trying to prevent re-refactoring after mentor's review ;)

Reply:
done